build_tools/travis/install.sh

```diff
@@ -13,15 +13,15 @@
 
 set -e
 
-# Fix the compilers to workaround avoid having the Python 3.4 build
-# lookup for g++44 unexpectedly.
-export CC=gcc
-export CXX=g++
-
 echo 'List files from cached directories'
 echo 'pip:'
 ls $HOME/.cache/pip
 
+export CC=/usr/lib/ccache/gcc
+# Useful for debugging how ccache is used
+# export CCACHE_LOGFILE=/tmp/ccache.log
+# ~60M is used by .ccache when compiling from scratch at the time of writing
+ccache --max-size 100M --show-stats
```

Comment:
I don't think it is needed. I certainly have come across SO answers that exactly say this, although nothing very authoritative.

side-comment: locally I don't need to set CC or CXX to have ccache working (you just add /usr/lib/ccache to PATH and that's it). I tried many different things to get it to work on Travis and that was one I found that was working.